zk/proofs/poc/src/verification_key/mod.rs

```diff
@@ -13,9 +15,15 @@ impl AsRef<Groth16PreparedVerificationKey> for PoCVerifyingKey {
 }
 
 pub static POC_VK: LazyLock<PoCVerifyingKey> = LazyLock::new(|| {
-    let vk_json = include_bytes!("verification_key.json");
+    let vk_path = circuits_utils::verification_key_path(CIRCUIT_NAME);
+    let vk_json = std::fs::read(&vk_path).unwrap_or_else(|error| {
+        panic!(
+            "Failed to read verification key from {}: {error}",
+            vk_path.display()
+        )
+    });
     let groth16_vk_json: Groth16VerificationKeyJsonDeser =
-        serde_json::from_slice(vk_json).expect("Key should always be valid");
+        serde_json::from_slice(&vk_json).expect("Key should always be valid");
     let groth16_vk: Groth16VerificationKey = groth16_vk_json
```

Comment:
This is wrong. There is a reason that we used `include_bytes` here and it is to avoid distributing external files if possible. The VKs should be part of the binary at the moment of building.